QModManager/API/SMLHelper/Assets/CustomFishPrefab.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace QModManager.API.SMLHelper.Assets
+{
+    using UnityEngine;
+    using QModManager.API.SMLHelper.Utility;
+    using Logger = QModManager.API.SMLHelper.Logger;
```

Comment:
Move usings before namespaces or keep them this way?

Reply:
@PrimeSonic I'd like to know your opinion

Reply:
I am always of the opinion that `using` statements are best kept inside the namespace.  
1. They can be simplified for namespaces inside the same assembly
2. They are harmless
3. Reduced need for aliases
4. For that 0.1% of cases where it will matter, getting into this habit will save you some disambiguation hell

Reply:
When adding the using statement automatically, VS adds it completely and doesn't detect that it could be simplified. This means that we get stuff like this:

```csharp
using QModManager.SMLHelper.Assets; // could be using Assets;
using Interfaces
```

and we have to manually edit it. 

Another argument is that simplified using statements are more likely to break when renaming (CTRL+R, CTRL+R) namespaces.

The only real difference is that they have higher priority over external namespaces, a thing which can be changed in `Code Editing > C# > Formatting Style > Namespace Imports`.

Reply:
This is not very important, but IMO they should be normalized, either per-project, or per-solution.

Reply:
I agree. They should absolutely be normalized, whichever approach we end up with.